setup.cfg

```
@@ -43,9 +43,17 @@ python_requires = >= 3.7
 [options.extras_require]
 dev =
     setuptools>=3.7.1
+pandas =
+    pandas>=1.3.5
 keepalive =
     keepalive>=0.5
 
 [options.entry_points]
 console_scripts =
     rqw = SPARQLWrapper.main:main
+
+[mypy]
+python_version = 3.7
+show_error_codes = True
+strict = True
+pretty = True
```

I would suggest adding `warn_unused_configs`, `warn_unused_ignores`, `warn_unreachable` also.

```suggestion
pretty = True
warn_unused_configs = True
warn_unused_ignores = True
warn_unreachable = True
```